### PR TITLE
adding: Home Page E-commerce files

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
image
image

Files to check:
* Inside Component folder all files.
* Home folder, Header Folder, Footer Folder.

Files to ignore: 
* All SCSS files.
* I'm implementing all CSS moves to SCSS files.

Feedback is highly appreciated :)

### CODE REVIEW

File: src/components/Header/subcomponents/HeaderMain.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+								</i>
+								<span>Wishlist</span>
+							</a>
+						</li>
+						<li className='li-tag-header card-header'>
+							<a className='a-tag-header-right' href='./checkout.html'>
+								<i className='fas fa-shopping-cart header-icon'>
+									{/* {itemCount ? (
+											<span className='cart-number'>{itemCount}</span>
+										) : null} */}
```
Comment (zoef-yt) [inline src/components/Header/subcomponents/HeaderMain.jsx]: Remove commented code.

File: src/components/Home/CarouselSlider/SubcarouselComponents/SlideContent.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const SlideContent = ({activeIndex, imageSlider}) => {
+	return (
+		<section>
+			{imageSlider &&
+				imageSlider.map((slide, index) => (
+					<div
+						key={index}
+						className={index === activeIndex ? "slides active" : "inactive"}>
+						<img className='slide-image' src={slide.urls} alt='' />
```
Comment (zoef-yt) [inline src/components/Home/CarouselSlider/SubcarouselComponents/SlideContent.jsx]: Don't keep the alt tag empty.

**Review by zoef-yt — APPROVED** [review]
The code looks good. The pr is too large for 1 review. Good to merge.

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import "./App.scss";
+import Header from "./components/Header/Header";
```
Comment (humzahasan) [inline src/App.jsx:2]: Suggestion : Have a common index file for all component and import from there. Something like this,
```js
import {Header,Footer,Home} from "./components/index"
```

File: src/components/Home/CarouselSlider/CarouselSlider.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import React, {useState, useEffect} from "react";
+import SlideContent from "./SubcarouselComponents/SlideContent";
```
Comment (humzahasan) [inline src/components/Home/CarouselSlider/CarouselSlider.jsx:2]: These can be clubbed into one import

File: src/components/Home/FeaturedBrands/FeatureBrands.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./FeaturedBrands.scss";
+import First from "../FeaturedBrands/BrandAssests/brand1.jpg";
+import Second from "../FeaturedBrands/BrandAssests/brand2.png";
+import Thrid from "../FeaturedBrands/BrandAssests/brand3.jpg";
+import Four from "../FeaturedBrands/BrandAssests/brand4.png";
```
Comment (humzahasan) [inline src/components/Home/FeaturedBrands/FeatureBrands.jsx]: A common import here too can help. Have all the images inside one index and import from there

File: src/components/Home/TopSelles/TopSells.jsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+				{productImages &&
+					productImages.map((brand, index) => (
+						<div key={index} className='product-container'>
+							<div className='product-image-container'>
+								<img
+									className='product-image'
+									src={brand.imageUrl}
+									alt='productimage'
+									loading='lazy'
+								/>
```
Comment (humzahasan) [inline src/components/Home/TopSelles/TopSells.jsx:37]: Lazy loading images! Good approach!

**Review by humzahasan — APPROVED** [review]
-  Good folder structure maintained. 
- Readable as well as easily understandable code.

Code looks good to merge! You can work on the suggestion for the next PR.

File: src/components/Header/subcomponents/HeaderMain.jsx (hunk 1 above)
Comment (santhoshpemmaka) [reply src/components/Header/subcomponents/HeaderMain.jsx]: I'm remove commented lines from file.

File: src/components/Home/CarouselSlider/SubcarouselComponents/SlideContent.jsx (hunk 2 above)
Comment (santhoshpemmaka) [reply src/components/Home/CarouselSlider/SubcarouselComponents/SlideContent.jsx]: I'm added text for alt attribute in the img tag. :)

File: src/App.jsx (hunk 3 above)
Comment (santhoshpemmaka) [reply src/App.jsx:2]: I'm changing for folder structure PR request. thanks 👍

File: src/components/Home/CarouselSlider/CarouselSlider.jsx (hunk 4 above)
Comment (santhoshpemmaka) [reply src/components/Home/CarouselSlider/CarouselSlider.jsx:2]: Okay, i'm changing one import statement.

File: src/components/Home/FeaturedBrands/FeatureBrands.jsx (hunk 5 above)
Comment (santhoshpemmaka) [reply src/components/Home/FeaturedBrands/FeatureBrands.jsx]: I'm convert all images import statement into index.js file. thanks :)

File: src/components/Home/TopSelles/TopSells.jsx (hunk 6 above)
Comment (santhoshpemmaka) [reply src/components/Home/TopSelles/TopSells.jsx:37]: Thanks :)

**Pull request merged**